packages/beacon-node/test/spec/presets/ssz_static.test.ts

```diff
@@ -56,9 +56,9 @@ const sszStatic =
       (ssz.bellatrix as Types)[typeName] ||
       (ssz.altair as Types)[typeName] ||
       (ssz.phase0 as Types)[typeName];
-    if (!sszType) {
-      throw Error(`No type for ${typeName}`);
```

Comment:
doesn't this just result in strange errors downstream if we don't throw here and exit the test run?

Reply:
Makes sense. I updated so that when `sszType` is null, associated tests are not run at all; also improved the error message to be more meaningfull.
Compared to the original `Error` based approach, a missing SSZ type will not prevent further tests to be run (the `Error` would short-circuit the logic)